languages/elixir/exercises/concept/try-rescue-else-after/.docs/introduction.md

```diff
@@ -0,0 +1,64 @@
+## Errors Expanded
+
+Using `try..rescue` is a powerful construct for catching errors when they occur in programs so that function can return sensible values if it is required. The `try..rescue` construct also offers us two additional features we can make use of:
+
+- the `else` block
+  - When the try block succeeds, the result is matched to this block.
+
+```elixir
+try do
+  :a
+rescue
+  _ -> :error
+else
+  :a -> :success
+end
+# => :success
+```
+
+- the `after` block
```

Comment:
You could consider moving this bit of text above the previous code example, and then merge the two examples into one.